tests/baselines/reference/unusedClassesinNamespace1.errors.txt

```diff
@@ -0,0 +1,12 @@
+tests/cases/compiler/unusedClassesinNamespace1.ts(3,11): error TS6133: Variable 'c1' has never been used.
+
+
+==== tests/cases/compiler/unusedClassesinNamespace1.ts (1 errors) ====
+    
+    namespace Validation {
+        class c1 {
+              ~~
+!!! error TS6133: Variable 'c1' has never been used.

```

Comment:
we probably should change the text for this error message to say `'C1` declared but never used.`, and avoid the calling it variable as it is confusing.